CONTRIBUTING.md

```diff
@@ -38,6 +38,8 @@ Other tools that are optional but useful:
 - [uv][]
 - [Make][]
 
+If you're using [Nix][] and [direnv][], everything you need is in the `shell.nix` file at the root of this repo, which will be automatically activated once you've entered the directory if you've run `direnv allow` at least once.
```

Comment:
Strictly speaking not *everything*. There may be tools that have dependencies not provided by `shell.nix`. In particular, I couldn't figure out what SciLean needs.

Reply:
Good point, I'll clarify that comment.